wolfcrypt/src/ecc.c

```diff
@@ -3993,30 +4000,40 @@ static int wc_ecc_new_point_ex(ecc_point** point, void* heap)
    int err = MP_OKAY;
    ecc_point* p;
 
+   WOLFSSL_ENTER("wc_ecc_new_point_ex");
    if (point == NULL) {
        return BAD_FUNC_ARG;
    }
 
    p = *point;
 #ifndef WOLFSSL_NO_MALLOC
    if (p == NULL) {
-      p = (ecc_point*)XMALLOC(sizeof(ecc_point), heap, DYNAMIC_TYPE_ECC);
+       WOLFSSL_MSG_EX("XMALLOC ecc_point %d bytes.", sizeof(ecc_point));
```

Comment:
Again this logging seems like debugging and should be removed.

Reply:
Hi @dgarske  - yes, the debugging breadcrumbs were included as I found them extremely valuable when enabling `DEBUG_WOLFSSL` to see where things were proceeding, and where code would quietly just halt when out of memory (in my case - the memory constrained embedded devices such as ESP32 or the tiny 32KB Arduino). 

I thought they might be helpful to a future developer in a similar situation.  The `TODO` was however, an oversight on my part and was not intended to be included.

Do you wish to have all of the breadcrumbs in this PR removed, or shall I remove the ones you mentioned and perhaps some others at my discretion?